### PR TITLE
Add delete images

### PR DESCRIPTION
**Pull request by edward-hayes**
Adds delete images command.
Adds IDs to the export-images command so that we can easily find image ids for the delete-images command

Also adds a few details to the export-images command to make a given image easier to recognize

### CODE REVIEW

**Comment by edward-hayes** [comment]
Needs https://github.com/vend/govend/pull/16 to be included before merge

File: commands/deleteImages.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	}
+
+	// Make the requests
+	count := 0
+	for _, id := range ids {
+		fmt.Printf("\nDeleting %v", id)
+		url := fmt.Sprintf("https://%s.vendhq.com/api/2.0/product_images/%s", DomainPrefix, id)
+		_, err = vendClient.MakeRequest("DELETE", url, nil)
+		if err != nil {
+			fmt.Printf(color.RedString("Failed to delete consignment: %v", err))
```
Comment (ObedSj) [inline commands/deleteImages.go:55]: Typo, "Failed to delete Product"

**Review by ObedSj — CHANGES_REQUESTED** [review]
Other than fixing the typo, it looks good to me 👍

File: commands/deleteImages.go (hunk 1 above)
Comment (edward-hayes) [reply commands/deleteImages.go]: lol

**Pull request merged**